components/Avatar/Avatar.css

```diff
@@ -18,8 +18,8 @@
     border-bottom-right-radius: 10px;
     width: 15.625rem;
     grid-area: aside;
-    height: 1800px;
```

Comment:
Try to use consistent units.

Reply:
Rest everything looks good to me.

Reply:
yea i have updated that thanks for reviewing